### ISSUE TITLE
Why are dev dependencies needed for the build process?

### ISSUE THREAD

**Issue by AntoniosProvidakis** [mars/heroku-cra-node#35]
Hello,

In the build script (see [here](https://github.com/mars/heroku-cra-node/blob/master/package.json#L10)) you call:
```
npm install --only=dev --no-shrinkwrap
```
which downloads the dev dependencies if I'm not mistaken.

Is it needed at all since this is a production build on Heroku?

Thanks

**Comment by mars** [comment]
Hi Antonios,

To answer your question, I reached for the Create React App `package.json` which by default installs its `react-scripts` package as devDependencies.

Well apparently, it used to do that! I found that [back in December the CRA update](https://github.com/mars/heroku-cra-node/commit/1afb09546e34ccb165d87daf0d71d0d13c0ffb15#diff-faf439330726cea53bf58d333745d2de) moved `react-scripts` to normal dependencies.

So this devDependencies hokey-pokey dance is no longer needed!

Thanks for bringing this up 😄🙌
